lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -286,8 +286,8 @@ module.exports = [
   },
   {
     lhr: {
-      requestedUrl: 'http://localhost:10200/perf/trace-elements.html?missing',
```

Comment:
This change doesn't seem necessary, or better?

Reply:
the test is all about DOM elements that are no longer in the DOM ("removed") **and** also that are not available to the protocol due to being evicted from some DOM node cache.

even if all this test was looking at was the first part, i'd still be lobbying for a rename from "missing" to "removed" as that's the more typical term.
but since this situation is more about REMOVED+EVICTED (which is why we're doing the rerenders) i'd like to be using the most clear term.

you can also see where i edited Patrick's PR description for more context.
https://github.com/GoogleChrome/lighthouse/pull/10877

(background: i was trying to understand the point of this test. i saw "missing" and concluded it was all about DOM elements just being removed from the DOM. that led to a buncha confusion)